### PR TITLE
Update philips.ts to support Hue GU5.3/MR16/LCG005

### PR DESCRIPTION
**Pull request by thk-socal**

### CODE REVIEW

**Comment by Koenkk** [comment]
Thanks!

File: src/devices/philips.ts
Comment (mundschenk-at) [inline src/devices/philips.ts:1]: I am a bit surprised that these would be separate models - all GU5.3 devices are in the MR16 form factor and they were just introduced, so it is doubtful that they have a new revision/hardware platform already.

Comment (thk-socal) [reply src/devices/philips.ts:1]: Well my MR16, here in the US, were LCG005 model number. Not sure why, but that is the model reported by zigbee2mqtt when joining them.

Comment (mundschenk-at) [reply src/devices/philips.ts:1]: Interesting. I think we should join the definitions though, no?

**Pull request merged**